File: bindings/js/src/generators/RAMLGenerator.ts
/**
 * Created by antoniogarrote on 05/05/2017.
 */

import {BaseGenerator} from "./BaseGenerator";
import {Clojure} from "../Clojure";

/**
 * Serialised the AMF model as a RAML YAML document
 */
export class RAMLGenerator extends BaseGenerator {
    protected generator(): any {
        return new Clojure.amf.__GT_RAMLGenerator();
    }
}

File: bindings/java/src/org/raml/amf/generators/RAMLGenerator.java
package org.raml.amf.generators;

/**
 * Created by antoniogarrote on 04/05/2017.
 */

/**
 * Serialised the AMF model as a RAML YAML document
 */
public class RAMLGenerator extends BaseGenerator {

    @Override
    protected String generatorConstructor() {
        return "->RAMLGenerator";
    }
}
